src/utils/shared.spec.js

```diff
@@ -134,10 +134,12 @@ describe('Shared methods and properties', () => {
   });
 
   it('should resolve a directory for Stache assets folder', () => {
-    let result = shared.resolveAssetsPath('foo');
+    let result = shared.resolveAssetsPath('foo')
```

Comment:
This test feels a little weird now.  I think instead I would do something like the following, which shows the `resolveAssetsPath` is OS specific.

```
const path = require('path');
const result = shared.resolveAssetsPath('foo');
expect(result.endsWith(path.join('src', 'stache', 'foo')).toBe(true);
```

Having said that, I think a more important test would be to show that the `buildGlobalDataFromJson` function supports a root of `src/stache/data` and `src\stache\data`.  Adding the same `replace` logic to the spec file doesn't validate that.